string id
float64 pointX
float64 pointY
float64 pointHA
float32 vehSpeed
